__tests__/propertyVolumeTests.ts

```diff
@@ -0,0 +1,21 @@
+import * as properties from '../src'
+import * as config from 'config'
+
+describe('Azure flex properties module should read from specified mount', () => {
+  test('should start have secrets read from volume', () => {
+    let config: any = {}
```

Comment:
if you aren't going to be reassigning a variable use `const`

Reply:
sure